browser-test/src/applicant/application_review.test.ts

```diff
@@ -9,9 +9,392 @@ import {
   testUserDisplayName,
   waitForPageJsLoad,
   validateScreenshot,
+  enableFeatureFlag,
 } from '../support'
 
 test.describe('Program admin review of submitted applications', () => {
+  test('all major steps with multiple file upload flag', async ({
```

Comment:
**question (non-blocking):** Is your idea, in copying all the major steps, that this test will replace the following test once the feature flag is removed?  For the moment, it seems like a lot of duplication, much of it unnecessary, but I would understand if it's going to make removing the flag easier.

Reply:
Understood.  I'll put this on our agenda for the next eng sync so that we can discuss a best approach.  I was unsure which approach to take on the feature I'm currently working on too.